test/Benchmarks/src/Vector/Distinct.enso

```diff
@@ -1,27 +1,33 @@
 from Standard.Base import all
 
-from Standard.Test import Bench
+from Standard.Test import Bench, Phase_Conf
 
 import project.Vector.Utils
 
+type Setup
+    Value random_vec uniform_vec random_text_vec uniform_text_vec
 
-options = Bench.options . size 100 . iter 20
+    setup : Setup
+    setup =
+        random_vec = Utils.make_random_vec 100000
+        uniform_vec = Vector.fill 100000 1
+        random_text_vec = random_vec.map .to_text
+        uniform_text_vec = random_vec.map .to_text
+        Setup.Value random_vec uniform_vec random_text_vec uniform_text_vec
 
-random_vec = Utils.make_random_vec 100000
-uniform_vec = Vector.fill 100000 1
-random_text_vec = random_vec.map .to_text
-uniform_text_vec = random_vec.map .to_text
+
+options = Bench.options . set_warmup (Phase_Conf.Iters 1) . set_measure (Phase_Conf.Iters 1) . set_setup Setup
 
 collect_benches = Bench.build builder->
-    builder.group "Vector Distinct" options group_builder->
-        group_builder.specify "Random Integer Vector Distinct" <|
-            random_vec.distinct
-        group_builder.specify "Uniform Integer Vector Distinct" <|
-            uniform_vec.distinct
-        group_builder.specify "Random Text Vector Distinct" <|
-            random_text_vec.distinct
-        group_builder.specify "Uniform Text Vector Distinct" <|
-            uniform_text_vec.distinct
+    builder.group "Vector_Distinct" options group_builder->
+        group_builder.specify "Random_Integer_Vector_Distinct" <|
+            arg -> arg.random_vec.distinct
+        group_builder.specify "Uniform_Integer_Vector_Distinct" <|
+            arg -> arg.uniform_vec.distinct
+        group_builder.specify "Random_Text_Vector_Distinct" <|
+            arg -> arg.random_text_vec.distinct
```

Comment:
I find this `arg -> arg` syntax ugly. Moreover we don't need it when we use the _lazy atom fields_ approach. Let us thus return back to simple `data.uniform_text_vec.distinct`.